### ISSUE TITLE
[FR] /riot/account/v1/active-shards/by-game/{game}/by-puuid/{puuid} for LoL

### ISSUE THREAD

**Issue by geozukunft** [RiotGames/developer-relations#836]
Now that League is using account-v1 as well it would be nice for league to get support on the active-shards endpoint that way it would no longer be needed to ask a player on which platform they are currently playing on. 

labels:
api: account-v1
scope: lol
topic: riot games api
type: feature request

**Comment by lucasMesquitaBorges** [comment]
I know it's a long time after this transition but we finally got unblocked on this.

The support for active shard for lol/tft was added, but it's a new endpoint!

https://developer.riotgames.com/apis#account-v1/GET_getActiveRegion

Feel free to re-open or create a new ticket if needed!
